phase3/Kconfig

```diff
@@ -40,4 +40,10 @@ config phase3.weave_net
 	  Should we deploy weave-net? You likely only want this if using kubeadm for phase2.
 endif
 
+config phase3.cni
+	string "CNI plugin to use"
+	default weave
```

Comment:
@pipejakob I will try to roll and spin up Jenkins jobs using these docs, will ping you when/if I get stuck :)